### PR TITLE
launch /academy

### PR DESCRIPTION
**Pull request by nylira**
Starting on the Academy page.

cosmos-academy-1

### CODE REVIEW

**Comment by nylira** [comment]
@adrianbrink yeah, I'll work on that.

**Comment by nylira** [comment]
@adrianbrink the page contents is now pulled from here: https://github.com/tendermint/aib-data/blob/develop/md/academy.md Please feel free to edit it to your liking.

**Comment by nylira** [comment]
New preview:

005 - academy - cosmos network - http___localhost_8700_academy

File: src/components/pages/PageDevelopersIndex.vue [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <template lang="pug">
-page(title="Developers" subtitle="Learn how to bootstrap your blockchain with Cosmos."): text-container
+page(title="Developers" subtitle="Learn about and contribute to the code that power the Cosmos Network."): text-container
```
Comment (jolesbi) [inline src/components/pages/PageDevelopersIndex.vue]: `powers`

**Comment by nylira** [comment]
Christine has approved the content, this PR is ready to go.

**Pull request merged**